src/Microsoft.Data.Analysis/ColumnArithmeticTemplate.ttinclude

```diff
@@ -88,6 +120,21 @@
         return false;
     }
 
+    // These are a subset of the implicit conversions allowed in C#. They are used to generate the return type for binary ops
+    public Dictionary<string, LinkedList<string>> primitiveTypeToImplicitConversions = new Dictionary<string, LinkedList<string>> {
+        {"sbyte", new LinkedList<string>(new List<string> {"int", "long", "float", "double", "decimal"})},
```

Comment:
Why `LinkedList` here? why not just `List<string>`? The order in `List<T>` is the priority order.

You may also consider exposing this as `IReadOnlyDictionary<string, IReadOnlyList<string>`. There's no reason this should ever need to change.

Reply:
It was `List<string>` before. I considered `LinkedList<string>` for a while when I wanted to change the order. Back to `List<string>` now.